src/data/shared/case-studies/case-studies-list.js

```diff
@@ -767,6 +1044,82 @@ const caseStudiesData = {
         },
       ],
     },
+    {
+      title: 'Line Coporation',
+      logo: LineCoporation,
+      description:
+        '<strong>Line Corporation</strong> uses eBPF at scale for load balancing and tracing in their private cloud',
+      links: [
+        {
+          text: 'Talk',
+          to: 'https://www.youtube.com/watch?v=cxfVpBYlol4',
+        },
+      ],
+    },
+    {
+      title: 'Exein',
+      logo: Exein,
+      description:
+        '<strong>Exein</strong> uses eBPF in their open source runtime security framework for IoT',
+      links: [
+        {
+          text: 'Talk',
+          to: 'https://youtu.be/vmRQXRit-sY?si=WurbrS-lNCEsDWGS',
+        },
+      ],
+    },
+    {
+      title: 'Ant Group',
+      logo: AntGroup,
+      description:
+        '<strong>Ant Group</strong> uses eBPF in their cloud native security paltform for vulnerability live-patching, and system anomaly detection',
+      links: [
+        {
+          text: 'Talk',
+          to: 'https://youtu.be/TXlxgDvxcoY?si=iAsbxdWJLlt2p9SV',
+        },
+      ],
+    },
+    {
+      title: 'Sysdig',
```

Comment:
is there a reason Sysdig is listed twice?

Reply:
fr-fr & en